Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:latest
+RUN apt-get update && apt-get install unzip make wget binutils-avr git -qy
+# COPY avr32studio-ide-2.6.0
+ARG TOOLCHAIN_LINK=https://cloud.nitrokey.com/s/a4Z2abXRjjYEzDJ/download
+RUN cd / && wget ${TOOLCHAIN_LINK} -O toolchain.zip -q && \
+    unzip toolchain.zip && \
```

Comment:
Can we make unzip silent? It clutters the view.

Reply:
done